src/core.js

```diff
@@ -632,21 +632,8 @@ define([
                     label: "Form Name",
                     slug: "formName"
                 },
```

Comment:
A single trailing comma is okay (in our non-IE world), it's multiples that get weird. I like trailing commas for reducing the number of lines you have to touch, though vellum uses both styles.